pydantic_openapi_helper/inheritance.py

```diff
@@ -304,6 +312,10 @@ def class_mapper(models, find_and_replace=None):
 
     # add enum classes to mapper
     schemas = get_schemas_inheritance(models)
+
+    missing_items = {k: v for k,
+                     v in mapper.items() if k not in schemas.keys()}
```

Comment:
This is probably more 'pythonic'!

```suggestion
    missing_items = {k: v for k,
                     v in mapper.items() if k not in schemas}
```